Extensions/stats.js

```diff
@@ -350,29 +350,32 @@ XKit.extensions.stats = new Object({
 
 		if (XKit.extensions.stats.window_id !== m_window_id) { return; }
 
+		var m_html = "";
 		if (blog_mode !== true) {
-			var m_html = "<div class=\"m_window_title\">Results for your dashboard</div>" +
+			m_html = "<div class=\"m_window_title\">Results for your dashboard</div>" +
 					"<div class=\"xkit-stats-separator\"><div>Top 4 blogs</div></div>" +
 					"<div class=\"xkit-stats-blog-list\">";
 		} else {
-			var m_html = "<div class=\"m_window_title\">Results for \"" + blog_url + "\"</div>" +
+			m_html = "<div class=\"m_window_title\">Results for \"" + blog_url + "\"</div>" +
 					"<div class=\"xkit-stats-separator\"><div>Top 4 blogs</div></div>" +
 					"<div class=\"xkit-stats-blog-list\">";
 		}
 
 		var m_count = 0;
 
-		for (var i=0;i<users.length;i++){
-			if (m_count == 4) {break; }
-			var perc = Math.round((users[i].count * 100) / posts.length);
-			var mx_html = 	"<a target=\"_BLANK\" href=\"http://" + users[i].url + ".tumblr.com/\"><div class=\"xkit-stats-blog\">" +
-						"<img src=\"https://api.tumblr.com/v2/blog/" + users[i].url + ".tumblr.com/avatar/32\" class=\"m_avatar\">" +
-						"<div class=\"m_title\">" + users[i].url + "</div>" +
+		users.forEach(function(user) {
+			if (m_count == 4) {
+				return;
```

Comment:
This (accidentally) ends up having the same behavior because m_count is never updated once it hits 4